pages/design-tokens/typesetting/overview.md

```diff
@@ -610,7 +612,7 @@ We use the `typeset()` mixin on all our components to get the effect of default
 #### Example: Settings and typesetting
 
 ```scss
-// in _uswds-theme-typography:
+// in _uswds-theme:
```

Comment:
Thinking it would be more clear to say `// in your settings configuration` instead of `//in _uswds-theme`.  I think it would also help to wrap the example settings with `@use "uswds-core" with (` like the user would in the settings config.  Also, the semi-colons should be commas. 

```
 @use "uswds-core" with (
  $theme-respect-user-font-size: true,
  $theme-font-type-sans:         'public-sans',
  $theme-font-role-ui:           $theme-font-sans,
  $theme-type-scale-sm:          5,
  $theme-body-font-size:         'sm',
  $theme-body-line-height:       5,
 )
```